externals/kyuubi-trino-engine/src/test/scala/org/apache/kyuubi/engine/trino/operation/TrinoOperationSuite.scala

```diff
@@ -71,6 +71,72 @@ class TrinoOperationSuite extends WithTrinoEngine with HiveJDBCTestHelper {
     }
   }
 
+  test("trino - get schemas") {
+    case class SchemaWithCatalog(catalog: String, schema: String)
+
+    withJdbcStatement() { statement =>
+      statement.execute("CREATE SCHEMA IF NOT EXISTS memory.test_escape_1")
+      statement.execute("CREATE SCHEMA IF NOT EXISTS memory.test2escape_1")
+      statement.execute("CREATE SCHEMA IF NOT EXISTS memory.test_escape11")
+
+      val meta = statement.getConnection.getMetaData
+      val resultSetBuffer = ArrayBuffer[SchemaWithCatalog]()
+
+      val schemas1 = meta.getSchemas(null, null)
+      while (schemas1.next()) {
+        resultSetBuffer +=
+          SchemaWithCatalog(schemas1.getString(TABLE_CATALOG), schemas1.getString(TABLE_SCHEM))
+      }
+      assert(resultSetBuffer.contains(SchemaWithCatalog("memory", "information_schema")))
+      assert(resultSetBuffer.contains(SchemaWithCatalog("system", "information_schema")))
```

Comment:
do trino-jdbc or trino-cli modles expose system tables/dbs to end-users?

Reply:
> do trino-jdbc or trino-cli modles expose system tables/dbs to end-users?

Yes, it's readable for end-users.